src/Parsers/GetObject/Multiple/Streamed.php

```diff
@@ -0,0 +1,93 @@
+<?php namespace PHRETS\Parsers\GetObject\Multiple;
+
+use GuzzleHttp\Psr7\Response;
+use PHRETS\Http\Response as PHRETSResponse;
+use Illuminate\Support\Collection;
+use PHRETS\Parsers\GetObject\Single;
+
+class Streamed
+{
+    // in bytes
+    // TODO: parameterize this
+    private const CHUNK_SIZE = 20 * 1000 * 1000;
+
+    public function parse(PHRETSResponse $response) : \Generator
+    {
```

Comment:
Maybe something like this?
```suggestion
    public const DEFAULT_CHUNK_SIZE = 20 * 1000 * 1000;

    public function parse(PHRETSResponse $response, ?int $chunkSize = null) : \Generator
    {
        $chunkSize ??= self::DEFAULT_CHUNK_SIZE;
```